shared/javascripts/network_service.js

```diff
@@ -437,15 +437,19 @@ var privlyNetworkService = {
    * Show/hide the appropriate navigation items for when the user is logged out.
    */
   showLoggedOutNav: function() {
-    
-    // Don't show the nav at all if the content is injected.
-    if(typeof privlyHostPage !== "undefined" && privlyHostPage.isInjected()) {
-      return;
+
+    if (privlyNetworkService.platformName() === "ANDROID") {
+      androidJsBridge.showLoginActivity();
+    } else {
+      // Don't show the nav at all if the content is injected.
+      if (typeof privlyHostPage !== "undefined" && privlyHostPage.isInjected()) {
+        return;
+      }
+      $(".logged_in_nav").hide();
+      $(".logged_out_nav").show();
+      $(".injected_hide").show();
+      privlyNetworkService.mobileHide();
```

Comment:
You can move this into the mobile block you defined since it only applies to mobile.